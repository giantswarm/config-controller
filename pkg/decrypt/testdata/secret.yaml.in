a:
  b:
    c: encrypted
key-0: encrypted
key-1: encrypted
