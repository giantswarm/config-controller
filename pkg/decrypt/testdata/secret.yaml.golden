a:
  b:
    c: decrypted
key-0: decrypted
key-1: decrypted
